packages/workflow/src/config/routes.ts

```diff
@@ -107,6 +108,15 @@ export const getRoutes = () => {
         tags: ['api'],
         description: 'Archive record endpoint'
       }
+    },
+    {
+      method: 'POST',
+      path: '/duplicate-record',
+      handler: duplicateRecordHandler,
+      config: {
+        tags: ['api'],
+        description: 'Duplicate record endpoint'
+      }
```

Comment:
Can we use the same url pattern as the others i.e. `/records/{recordId}/mark-duplicate` please?